pkg/operators/wasm/testdata/badguest/program.go

```diff
@@ -256,6 +286,38 @@ func gadgetInit() int {
 	assertNonZero(setConfig(invalidStrPtr, stringToBufPtr("value"), uint32(api.Kind_String)), "setConfig: bad key ptr")
 	assertNonZero(setConfig(stringToBufPtr("key"), invalidStrPtr, uint32(api.Kind_String)), "setConfig: bad value ptr")
 
+	/* Map */
+	assertZero(getMap(0), "getMap: bad map pointer")
```

Comment:
nit: use invalidStrPtr instead of 0 as 0 is handled in a special way: https://github.com/inspektor-gadget/inspektor-gadget/blob/4e50955197ed82a2a47ac67cbfbe844c6d56b351/pkg/operators/wasm/helpers.go#L61-L64

Same for pointers.